src/Aspire.Hosting/Dashboard/DashboardLifecycleHook.cs

```diff
@@ -214,15 +216,55 @@ private void ConfigureAspireDashboardResource(IResource dashboardResource)
                 }
             }
 
-            // Configure frontend browser token
-            if (!string.IsNullOrEmpty(browserToken))
+            // Configure frontend auth
+            context.EnvironmentVariables[DashboardConfigNames.DashboardFrontendAuthModeName.EnvVarName] = dashboardAuthMode.ToString();
+
+            if (dashboardAuthMode == FrontendAuthMode.OpenIdConnect)
             {
-                context.EnvironmentVariables[DashboardConfigNames.DashboardFrontendAuthModeName.EnvVarName] = "BrowserToken";
-                context.EnvironmentVariables[DashboardConfigNames.DashboardFrontendBrowserTokenName.EnvVarName] = browserToken;
+                if (options.OpenIdConnect != null)
```

Comment:
Forwarding configuration from the host to the dashboard seems to be a common thing to want to do. Are there any plans to do this in a more structured, organized, reusable way?

Reply:
Feels like we should build something that lets you generically forward settings to the dashboard (like a configuration section) where dashboard config can be specified.

Reply:
I agree this is a friction point.  

In this case the prefix is known and I don't think that Aspire should care what settings you use, if it's configurable for a normal app this way then I'd expect it to just pass it through here as well without needing to manually wire it all up.  Unfortunately, I was not able to come up with a way to achieve that without the POCO that limits the options and creates a model that now needs to be maintained to stay in sync with the actual package.  

I felt the same way about the FrontendAuthMethod enum as well, but that one is a little different because of the thin automation layer to automatically configure BrowserToken by default which seems appropriate for the majority of local dev scenarios.